pymc4/distributions/tensorflow/continuous.py

```diff
@@ -0,0 +1,97 @@
+"""
+PyMC4 continuous random variables.
+
+Wraps selected tfp.distributions (listed in __all__) as pm.RandomVariables.
+Implements random variables not supported by tfp as distributions.
+"""
+import tensorflow_probability as tfp
+tfd = tfp.distributions
+
+
+from .distribution import ContinuousDistribution
+
+__all__ = ['Normal']
+
+
+class Normal(ContinuousDistribution):
```

Comment:
We can try to avoid copy-pasting docs, probably with InheritDocs metaclass

Reply:
or this simpler solution https://stackoverflow.com/a/2025599

Reply:
Cool solution! Reduces a lot of copy-paste in class docstrings. I implemented the simpler verision
```
__doc__ = Distribution.__doc__ + __doc__
```
Although it seems to work only for class docstrings, not for individual methods. Also requires that base class docstring is not None.